usr/src/cmd/zpool/zpool_main.c

```diff
@@ -47,6 +48,7 @@
 #include <zone.h>
 #include <zfs_prop.h>
 #include <sys/fs/zfs.h>
+#include <sys/vdev_impl.h>
```

Comment:
It's possible that this is causing the compilation (lint) error (see http://jenkins.open-zfs.org/blue/organizations/jenkins/openzfs%2Fopenzfs/detail/PR-424/4/pipeline).  Is this needed just for `VDEV_LABELS`?  Maybe we should move that to a different header file (e.g. zfs.h)